src/test/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApClientTPCTest.java

```diff
@@ -31,15 +33,21 @@ public class MeasurementBasedApClientTPCTest {
 
 	/** Create a device state object containing the given parameters. */
 	private State createState(
```

Comment:
I have another PR editing some of this stuff. Let me look into what you did and we can discuss how to resolve on Mon.